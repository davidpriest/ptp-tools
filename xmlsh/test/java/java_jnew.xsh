# 
# Test of jnew 
# 

# Integer
i=jnew(java.lang.Integer 1)
xtype $i
echo $i

# Date
date=jnew(java.util.Date)
xtype $date



