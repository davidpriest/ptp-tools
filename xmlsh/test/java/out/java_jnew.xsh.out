java.lang.Integer
1
java.util.Date
